packages/passport-interface/src/RequestTypes.ts

```diff
@@ -158,3 +151,39 @@ export interface FeedRequest<T extends PCDPackage = PCDPackage> {
 export interface FeedResponse {
   actions: PCDAction[];
 }
+
+export interface PCDpassUserJson {
+  superuserEventConfigIds: string[];
+  uuid: string;
+  commitment: string;
+  email: string;
+  salt: string | null;
+}
+
+export interface ZupassUserJson {
+  email: string;
+  name: string;
+  role: ZuzaluUserRole;
+  visitor_date_ranges?: DateRange[] | null;
+  uuid: string;
+  commitment: string;
+  salt: null; // Zupass users never have a password salt
+}
+
+export interface ConfirmationEmailResponse {
+  token: string;
+}
+
+export interface SaltResponse {
+  salt: string | "";
+}
+
+export type VerifiedTokenResponse =
+  | {
+      verified: true;
+      error?: undefined;
```

Comment:
Is this different from `error: undefined` or omitting the `error` field entirely?

Reply:
in `js`, accessing a field that doesn't exist on an object results in `undefined`. the question mark makes a field altogether optional, whereas defining the type of a field to be `undefined` means that field actually *is* on the object, just that its value is `undefined`